docs/csharp/how-to/safely-cast-using-pattern-matching-is-and-as-operators.md

```diff
@@ -25,7 +25,7 @@ The preceding sample demonstrates other features of pattern matching to use with
 
 The preceding sample also shows how you use the type pattern in a `switch` statement where the variable may be one of many different types.
 
-If you want to test if a variable is a given type, but not assign it to a new variable, you can use the `is` and `as` operators for reference types and nullable types. The following code shows how to use the `is` and `as` statements that were part of the C# language before pattern matching was introduced to test if a variable is of a given type:
+If you want to test if a variable is a given type, but not assign it to a new variable, you can use the `is` and `as` operators for reference types, nullable reference types, and nullable value types. The following code shows how to use the `is` and `as` statements that were part of the C# language before pattern matching was introduced to test if a variable is of a given type:
```

Comment:
Since "reference types" includes "nullable reference types," is it necessary to add the latter phrase?

Reply:
I wasn't sure. I think it's better to remove it.